libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -59,12 +59,12 @@ interface MapboxNativeNavigator {
      * and verify that the user is still on the route. This method also determines
      * if an instruction needs to be called out for the user.
      *
-     * @param date the point in time to receive the status for.
+     * @param timeNano the point in time to receive the status for. Unit is nano seconds
      *
      * @return the last [TripStatus] as a result of fixed location updates. If the timestamp
      * is earlier than a previous call, the last status will be returned. The function does not support re-winding time.
      */
-    suspend fun getStatus(date: Date): TripStatus
+    suspend fun getStatus(timeNano: Long): TripStatus
```

Comment:
```suggestion
    suspend fun getStatus(timeNanos: Long): TripStatus
```
nit: we use `nanos` and `nano`, let's choose one of them

it's not a part of the PR, but `interface Time` has
```
fun nanoTime(): Long
fun millis(): Long
```
maybe rename `nanoTime()` to `nanos()` ?

Reply:
it's from System.nanoTime() so let's keep it as in java.
Gonna rename `timeNano` -> `nanoTime`

Reply:
done